lib/rsgem/tasks/create_gem.rb

```diff
@@ -34,6 +35,20 @@ def shell_command
           bundler_options
         ].compact.join(' ')
       end
+
+      def prepend_frozen_string_command
+        gem_name = context.gem_name
+
+        [
+          "lib/#{gem_name}.rb",
+          "lib/#{gem_name}/version.rb",
+          'spec/spec_helper.rb',
+          "spec/#{gem_name}_spec.rb"
+        ].map { |file_name| "#{gem_name}/#{file_name}" }
+          .map do |file_path|
+            "echo \"# frozen_string_literal: true\\n\\n$(cat #{file_path})\" > #{file_path}"
+          end.join(' && ')
+      end
```

Comment:
How about we create a task object for this? (like the ones here https://github.com/rootstrap/rsgem/tree/master/lib/rsgem/tasks)